.github/workflows/molecule_tests.yml

```diff
@@ -0,0 +1,37 @@
+---
+# SPDX-license-identifier: Apache-2.0
+##############################################################################
+# Copyright (c) 2021
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Apache License, Version 2.0
+# which accompanies this distribution, and is available at
+# http://www.apache.org/licenses/LICENSE-2.0
+##############################################################################
+
+name: Check Molecule Tests
+# yamllint disable-line rule:truthy
+on: [push, pull_request]
+
+jobs:
+  check-molecule:
```

Comment:
Is it possible to limit the running of this GH Action to just those PRs that impact its success? Or is that **everything**?

If for example it is primarily validating ansible recipes, then maybe filtering only on changes to `/lte/gateway/deploy`? We now use a GH action for path filtering - if possible.  First you declare a `path_filter` step like [this one](https://github.com/magma/magma/blob/master/.github/workflows/gcc-problems.yml#L22-L32).  Then you add a `needs:` and `if:` clause to your `check-molecule` [like this](https://github.com/magma/magma/blob/master/.github/workflows/gcc-problems.yml#L35-L37).

This drastically accelerates CI results for most PRs that don't impact the test - and enables setting this workflow as `required` in Github (you cannot mark `required` if you use Github's native path based filtering, sadly).